pupil-api/src/App.ts

```diff
@@ -27,10 +29,40 @@ class App {
     this.express = express()
     this.middleware()
     this.routes()
-
+    this.featureToggles()
     appInsights.startInsightsIfConfigured().catch(e => logger.error(e))
   }
 
+  private featureToggles (): void {
+    /**
+     * Load feature toggles
+     */
+    logger.info('ENVIRONMENT_NAME : ' + config.Environment)
+    const environmentName = config.Environment
+    let featureTogglesSpecific: string
+    let featureTogglesDefault: string
+    let featureTogglesSpecificPath: string
+    let featureTogglesDefaultPath: string
+    try {
+      featureTogglesSpecificPath = './feature-toggles/feature-toggles.' + environmentName
+      featureTogglesSpecific = environmentName ? require(featureTogglesSpecificPath) : null
+    // tslint:disable-next-line: no-empty
+    } catch (ignore) {} // missing feature files throw intentionally
+
+    try {
+      featureTogglesDefaultPath = './feature-toggles/feature-toggles.default'
+      featureTogglesDefault = require(featureTogglesDefaultPath)
+    // tslint:disable-next-line: no-empty
+    } catch (ignore) {} // missing feature files throw intentionally
+
+    const featureTogglesMerged = R.mergeRight(featureTogglesDefault, featureTogglesSpecific)
```

Comment:
Nothing to do with this PR but this merging of the feature toggles seems unnecessary.   Most people add new feature toggles to every config file rather than expecting merge behaviour where more specific config options overwrite the defaults.

Perhaps it would be better just to look for the feature toggle files in the order of most specific to least specific for a particular environment?

Reply:
that's the implementation taken verbatim from admin.  we should only have one file for a particular environment, and then a default, with them all in.

happy to review as a separate piece of work